content/docs/command-reference/params/index.md

```diff
@@ -32,6 +32,9 @@ organized as a tree hierarchy inside, as DVC will locate param names by their
 tree path. parameters files have to be manually written, or generated, and these
 can be versioned directly with Git.
 
+> Note that YAML files including `params.yaml` are expected to be in YAML 1.2
```

Comment:
let's embed this into the paragraph above the same way as you did this in other place? (I don't like when we put a lot of notes, it'll very fast get our of hands)

Reply:
Done.